sdk/eventhub/event-hubs/src/eventDataBatch.ts

```diff
@@ -153,7 +146,7 @@ export class EventDataBatchImpl implements EventDataBatch {
   /**
    * List of 'message' span contexts.
    */
-  private _spanContexts: SpanContext[] = [];
+  private _spanContexts: TracingContext[] = [];
```

Comment:
We now rely on `TracingContext` as an opaque type that holds a span, and our OTel package knows how to fish it out

Reply:
Either name is fine as they are not on public surface.